stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/ui/common/feature/messages/list/MessageListController.kt

```diff
@@ -1873,6 +1875,32 @@ public class MessageListController(
         })
     }
 
+    /**
+     * Block a user. Unlike ban the block is not channel related but rather directly to the user.
+     *
+     * @param userId the id of the user that will be blocked.
+     */
+    public fun blockUser(userId: String) {
+        chatClient.blockUser(userId).enqueue(onError = { error ->
+            onActionResult(error) {
+                ErrorEvent.BlockUserError(it)
+            }
+        })
+    }
+
+    /**
+     * Unblock a user.
+     *
+     * @param userId the id of the user that will be unblocked.
+     */
+    public fun unblockUser(userId: String) {
+        chatClient.unblockUser(userId).enqueue(onError = { error ->
+            onActionResult(error) {
+                ErrorEvent.BlockUserError(it)
```

Comment:
Should it be `ErrorEvent.UnblockUserError(it)`

Reply:
Its the same for Ban and Unban